blueprint/security/hybrid-system-security-plan.md

```diff
@@ -274,7 +275,7 @@ Outsourcing is applicable to the HybridSystem as it leverages both Azure and Off
 
 #### HybridSystem compliance approach
 
-The HybridSystem leverages Microsoft Azure and Office 365, which have been IRAP assessed at PROTECTED by Shearwater. With the exception of Microsoft Defender ATP, all HybridSystem services have been IRAP assessed.
+The HybridSystem leverages Microsoft Azure and Office 365, which have been IRAP assessed at PROTECTED by CyberCX. With the exception of Microsoft Defender ATP, all HybridSystem services have been IRAP assessed.
```

Comment:
I think you did it with the cloud native SSP: remove Defender ATP exception

Reply:
Corrected and updated